opsgenie/resource_opsgenie_role.go

```diff
@@ -0,0 +1,220 @@
+package opsgenie
+
+import (
+	"context"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/validation"
+	"log"
+
+	"github.com/opsgenie/opsgenie-go-sdk-v2/custom_user_role"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+)
+
+var validRights = []string{
```

Comment:
can you give this a more understandable name since this going to be global such as validCustomRolesRights

Reply:
yes, ofcourse 👍